packages/backend/src/helpers/flow-templates.ts

```diff
@@ -35,6 +46,68 @@ function validateAppAndEventKey(step: TemplateStep, templateName: string) {
   }
 }
 
+// This replaces the keys of columns which are names to the column ids for row data
+// e.g. { Rating: 5 } --> { column_id: 5 }
+function replaceRowData(
+  rowData: IJSONObject[],
+  columnNameToIdMap: Record<string, string>,
+) {
+  const replacedRowData: IJSONObject[] = []
+  for (const row of rowData) {
+    const newRow: IJSONObject = {}
+    for (const key in row) {
+      if (!columnNameToIdMap[key]) {
+        throw new Error(
+          'Creation of tile ran into an issue, please contact support@plumber.gov.sg',
```

Comment:
keep error messages concise for system logging in internal functions. Then try catch and return a standardized error message in the resolver

Reply:
thanks sir